### ISSUE TITLE
Working with .out files with rLPJGUESS package

### ISSUE THREAD

**Issue by halima1993** [biometry/RLPJ#50]
Greetings! I am a user of DGVMTools and devtools. I recently discovered rLPJGUESS package. Basically I have files that are in the format of .out files. I want to carry out sensitivity analysis by using these files in this package. My question is that can rLPJGUESS work with files that are in .out format?

P.s.  I have files related to different variables of three CMIP5 models (eg lai.out, co2.out) .

**Comment by florianhartig** [comment]
Which LPJ-GUESS version are you running?

**Comment by halima1993** [comment]
do you mean rLPJGUESS? I'm using rLPJGUESS version 1.1.0

**Comment by florianhartig** [comment]
No, I mean LPJ-GUESS - in principle, rLPJGUESS can read lai output (see supplementary in https://www.sciencedirect.com/science/article/pii/S1364815217310629?via%3Dihub, where we provide code to do a SA using LAI), but format of input and output files changes between versions, and many working groups have custom output files.

The current stable version of rLPJGUESS is designed for LPJ-GUESS v3.1. If you are running a different LPJ-GUESS version, you may have to adapt the rLPJGUESS functions. We are currently working on adaptations for 4.0 and 4.1, but also this will only work if you use one of the standard versions.
